tools/lkl/Makefile

```diff
@@ -48,6 +48,8 @@ export CFLAGS += -I$(OUTPUT)/include -Iinclude -Wall -g -O2 -Wextra \
 	 -Wno-unused-parameter \
 	 -Wno-missing-field-initializers -fno-strict-aliasing
 
+export CXXFLAGS += $(CFLAGS) -Wno-address-of-temporary
```

Comment:
Can this be moved to fuzzers/binder/Build?

Reply:
Done.